lib/sensu-plugins-aws/common.rb

```diff
@@ -17,8 +17,8 @@
 #
 
 module Common
-  def initialize
-    super()
+  def initialize(argv = ARGV)
+    super(argv)
```

Comment:
even though i chose not to do that route, adding this in allows us to spin up a new process and run the script for testing the output and exit code.

Reply:
I actually just mentioned forking a process for better testing.